src/index.html

```diff
@@ -9,6 +9,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="http://moyo.ua" class="img_container">
+        <img src="images/logo.png" alt="logo_icon">
+      </a>
+      <nav class="brands_container">
+        <a href="https://www.moyo.ua/foto_video/video_tech/" class="brand">ВИДЕО</a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="https://www.moyo.ua/foto_video/video_tech/" class="nav__link is">ВИДЕО</a>
```
Тут краще використовувати таку структуру. Адже це не що інше, як список